lib/validateSignature.ts

```diff
@@ -1,10 +1,8 @@
-import xmlcrypto from 'xml-crypto';
+import { SignedXml } from 'xml-crypto';
+import { select } from 'xpath';
```

Comment:
Shouldn't we add `xpath` as a dependency to `package.json`

Reply:
It's included in xml-crypto but they don't seem to import it anymore.